bounties/0052 Shopify NFT App.md

```diff
@@ -0,0 +1,66 @@
+---
+ID: "0052"
+Discussion: https://github.com/XRPLBounties/Proposals/discussions/0052
+Category: eCommerce App
+Status: Open
+Amount: $15,000
+---
+
+<!-- Please update this title -->
+
+# Shopify NFT App
+
+## Overview
+
+<!--
+Please provide the context required to complete the bounty.
+
+Questions you should answer here:
+1. What is the high level explanation of this bounty? (1-3 sentences)
+2. What problem is this solving?
+3. What are the requirements for this solution?
+-->
+
+There are over 2.5 million merchants that use the Shopify platform. All of which use marketing in some form or another to drive sales. 
+
+The goal of this app will be to give merchants a way to increase engagement through the use of NFTs. The actual marketing method is left up to the developer and the only requirement for this bounty is to use NFTs to incentivize customers to make further purchases or to increase engagement with the brand.
+
+Example:
+A customer would claim a random NFT via a QR code on their physical receipt or a link on their order confirmation page. Then upon collecting a certain set of NFT's that customer would trade them in for a discount or a free product.
+
+Such an app would allow merchants to use NFTs in their business, strengthen their communities and explore what Web3 has to offer. It would also expose them to  the speed and efficiency of using XRPL.
+
+
+
+## Milestones
+
+<!--
+Please split the bounty into smaller milestones with individual awards in the following template.
+The first milestone should be the core functionality, while the rest can be useful add-ons.
+
+| # | High-Level Description | Details | Proposed Potential Award |
+| 1 | ... | ... | $... |
+
+(The proposed amounts from milestones should add up to the amount listed at the top of the bounty proposal)
+-->
+
+| # | Description | Details | Proposed Potential Award |
+| --- | --- | --- | --- |
+|1| Create | Create a Shopify app that uses XLS-20 NFTs on XRPL to drive sales and engagement for a merchant. | $5,000 |
```

Comment:
is this meant to be open-sourced? Should the codebase be public? if so we should specify it

Reply:
It's not meant to be open source.

Reply:
Why not?

Reply:
They should be able to make money exclusively from the app, otherwise there's no incentive to continue supporting the app. People have tried to do open source apps but they've all failed due to other apps just copying their code and making money off their repo.

We can put in a component that would include tutorials on the general approach they took, or a general how to build an NFT ecommerce app. This part could be open source, but diverges a bit from the main goal.